azext_iot/central/commands_device_template.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from knack.util import CLIError
+
+from azext_iot.common import utility
+from .providers import CentralDeviceTemplateProvider
+
+
+def get_device_template(
+    cmd, app_id: str, device_template_id: str, central_dns_suffix="azureiotcentral.com"
+):
+    provider = CentralDeviceTemplateProvider(cmd, app_id)
+    return provider.get_device_template(
+        device_template_id=device_template_id, central_dns_suffix=central_dns_suffix
+    )
+
+
+def list_device_templates(cmd, app_id: str, central_dns_suffix="azureiotcentral.com"):
+    provider = CentralDeviceTemplateProvider(cmd, app_id)
+    return provider.list_device_templates(central_dns_suffix=central_dns_suffix)
+
+
+def map_device_templates(cmd, app_id: str, central_dns_suffix="azureiotcentral.com"):
+    provider = CentralDeviceTemplateProvider(cmd, app_id)
+    return provider.map_device_templates(central_dns_suffix=central_dns_suffix)
+
+
+def create_device_template(
+    cmd,
+    app_id: str,
+    device_template_id: str,
+    content: str,
+    central_dns_suffix="azureiotcentral.com",
+):
+    if not type(content) == str:
```

Comment:
You should use `isinstance` for type checks. This may not be needed though for `str` because the default arg type of a Knack argument should be `str`.